jumpscale/data/encryption/mnemonic.py

```diff
@@ -0,0 +1,133 @@
+import os
+import hashlib
+
+from .exceptions import FailedChecksumError
+
+
+wordlist = []
+
+with open("jumpscale/data/encryption/wordlist.txt", "r") as f:
```

Comment:
no hardcoded paths.

Reply:
Used os.path.dirname(__file__) to retrieve the module's directory instead of the path.